tests/clar_libgit2.c

```diff
@@ -222,7 +226,9 @@ git_repository *cl_git_sandbox_reopen(void)
 		git_repository_free(_cl_repo);
 		_cl_repo = NULL;
 
-		cl_git_pass(git_repository_open(&_cl_repo, _cl_sandbox));
+		cl_git_pass(git_repository_open(
+			&_cl_repo,
+			cl_fixture_basename(_cl_sandbox)));
```

Comment:
Why is this change necessary?

Reply:
When I was writing the tests for this patch, I wanted to have a single repository which I opened both in a bare configuration, without a working directory, and as a non-bare repository. To do this I tried to use `cl_git_sandbox_init`. Unfortunately, that code doesn't currently handle "nested/paths/to/fixtures". The clar fixture code copies just the leaf node of the path into the sandbox directory (e.g. just the `.gitted` folder), but `cl_git_sandbox_init` changes directory to the sandbox directory, and tries to open the full fixture path as a repository.

This change makes `cl_git_sandbox_init` use only the leaf name of the fixture's path when determining what the path will be relative to the sandbox.